sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/DatabasePersistence.java

```diff
@@ -200,6 +206,11 @@ public boolean onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                 db.execSQL("ALTER TABLE " + TABLE + " ADD COLUMN `" + COLUMN_PRIORITY + "` INTEGER DEFAULT " + PERSISTENCE_NORMAL);
                 return true;
             }
+
+            @Override
+            public void onOpen(SQLiteDatabase db) {
+                db.execSQL("CREATE INDEX IF NOT EXISTS " + INDEX_PRIORITY + " ON " + TABLE + " (" + COLUMN_PRIORITY + ")");
```

Comment:
Can you create index in onCreate and onUpgrade instead of doing this every time when it opens database?
I know it will only create the index if it doesn't exist but the method call itself isn't necessary to be called multiple times on same DB version.

Reply:
I suggest adding onCreate to the Listener and use add index in both callbacks.

Reply:
We don't have to update version as this was never released.